docs/tools/agentctl.md

```diff
@@ -2,21 +2,454 @@
 
 ---
 
-The agentctl should provide single universal tool for managing Ligato agents.
+## Intro
+
+Agentctl is a command line client to manage not only vpp-agent, but any agent built with Ligato framework.
 
 ![agentctl](../img/tools/agentctl.png)
 
-## Goals
+**Table of contents:**
+
+- [Intro](#intro)
```

Comment:
It looks weird to put the table of contents under a section listed in the table of contents.